mcp-wrapper/mcp_server.py

```
@@ -1,151 +1,144 @@
+# Add the src directory to the Python path to import ASR-GoT modules
+current_dir = Path(__file__).parent
+src_dir = current_dir.parent / "src"
+sys.path.insert(0, str(src_dir))
+
+# Lazy import ASR-GoT modules to speed up startup
+ASRGoTProcessor = None
+ASRGoTGraph = None
+
+def _import_asr_got():
+    """Lazy import ASR-GoT modules when needed."""
+    global ASRGoTProcessor, ASRGoTGraph
```

Pylint (3.3.7)

[warning] Using the global statement

(W0603)